src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2342,6 +2342,38 @@
      * A list of resources in the Java 7 try-with-resources construct.
      * This is a child of RESOURCE_SPECIFICATION.
      *
+     * <p>For example:</p>
+     * <pre>
+     *     try (FileReader fr = new FileReader("config.xml")) {}
```

Comment:
I do not see finally here

Reply:
@romani I have added finally. Please review this.